pkg/cmd/cli/cmd/edit.go

```diff
@@ -119,14 +121,20 @@ func RunEdit(fullName string, f *clientcmd.Factory, out io.Writer, cmd *cobra.Co
 		return err
 	}
 
-	r := b.Flatten().Do()
 	infos, err := r.Infos()
```

Comment:
You don't need to check err above if you call infos and check the error here.  It's the same error.